ui/client/containers/Auth/strategies/RemoteAuthStrategy/externalAuthModule.d.ts

```diff
@@ -0,0 +1,47 @@
+import {ComponentType, PropsWithChildren} from "react"
```

Comment:
I think we need some comments regarding:
- that this interface can be implemented by external modules
- short description how workflow for externalAuthModule works, what is possible etc.